tmt/steps/prepare/distgit.py

```diff
@@ -264,7 +266,7 @@ def go(
                 cwd=source_dir,
             )
         except tmt.utils.RunError as error:
-            raise tmt.utils.PrepareError("Unable to 'rpmbuild -bp'.", causes=[error])
+            raise tmt.utils.PrepareError("Unable to 'rpmbuild -bp'.", causes=[error]) from error
```

Comment:
```suggestion
            raise tmt.utils.PrepareError("Unable to 'rpmbuild -bp'.") from error
```
I believe we can drop `causes` in this case? @happz